scripts/ec2cleanup.py

```diff
@@ -218,8 +220,29 @@ def delete_queues(provider_mgmt, excluded_queues, output):
         logger.exception('Exception in %r', delete_queues.__name__)
 
 
+def delete_s3_buckets(provider_mgmt, bucket_name, output):
+    bucket_list = []
+    provider_name = provider_mgmt.kwargs['name']
+    try:
+        for bucket in provider_mgmt.list_s3_bucket():
+            if bucket_name not in bucket:
+                logger.info("Bucket name '%s' not found in list of buckets", bucket_name)
```

Comment:
Required:
You are pointing `bucket_name` means `default="smartstate"` or something that you pass. It's creating conflict against your statement.
I would like to suggest rather than logging each `bucket`. Please log `info` when not single button found in fetched buckets from `ec2` like `smartstates bucket not  found`